src/main/scala/org/broadinstitute/dsde/firecloud/utils/StreamingPassthrough.scala

```diff
@@ -10,7 +10,7 @@ import akka.http.scaladsl.server.directives.{BasicDirectives, RouteDirectives}
 import akka.stream.scaladsl.{Sink, Source}
 import com.google.common.net.UrlEscapers
 import com.typesafe.scalalogging.Logger
-import org.broadinstitute.dsde.firecloud.FireCloudExceptionWithErrorReport
+import org.broadinstitute.dsde.firecloud.{FireCloudExceptionWithErrorReport}
```

Comment:
This looks leftover from a different set of imports, these braces should be removed.